CMakeLists.txt

```diff
@@ -17,9 +17,10 @@ find_package(message_filters REQUIRED)
 find_package(tf2_ros REQUIRED)
 find_package(laser_geometry REQUIRED)
 find_package(angles REQUIRED)
-find_package(xmlrpcpp REQUIRED)
 find_package(rostime REQUIRED)
 
+find_package(Eigen3 REQUIRED)
```

Comment:
@bfjelds is this only needed for the laser_geometry headers? If so, should that package export the necessary include directories?

Reply:
that makes more sense!  i feel like i'm learning as i go with ament ... will try exporting from laser_geometry.